lib/registry.js

```diff
@@ -48,9 +57,9 @@ class Registry {
 		return acc;
 	}
 
-	metrics() {
+	metrics(opts) {
 		return this.getMetricsAsArray()
-			.map(this.getMetricAsPrometheusString, this)
+			.map(metric => this.getMetricAsPrometheusString.bind(this)(metric, opts))
```

Comment:
Don't use bind, just call it normally. The arrow function ensures correct `this`